app/helpers/is-modified-longer-then-6-months.js

```diff
@@ -0,0 +1,19 @@
+import { helper } from '@ember/component/helper';
+import { format, isBefore } from 'date-fns';
+
+export function isModifiedWithinLast6Months([modifiedDate]) {
+  const sixMonthsAgo = new Date();
+  sixMonthsAgo.setMonth(sixMonthsAgo.getMonth() - 6);
+  const today = new Date();
+  let result = {};
+  if (format(today, 'dd-MM-yyyy') === format(modifiedDate, 'dd-MM-yyyy')) {
```

Comment:
**Suggestion:** This check causes an error when `modifiedDate` is undefined. As a result going to some random organisations (Provincie Antwerpen and Gemeente Hasselt) crashed the frontend with an error in the developer console. Might I suggest adding an extra check such that the function falls back to the last `else` in such cases instead of causing a crash (better suggestions are fine as well):

```suggestion
  if (
    modifiedDate &&
    format(today, 'dd-MM-yyyy') === format(modifiedDate, 'dd-MM-yyyy')
  ) {
```

Note, I suspect that at some point a default modified date will be added to the DEV data, and this error should theoretically no longer occur with "correct" data. But considering the minor impact an extra check has on the code I would argue it is worthwhile to add in this to prevent potential user-facing crashes.